README.md

```diff
@@ -139,14 +139,34 @@ Supported boards are: feather_nrf52840_express feather_nrf52840_express pca10056
 Makefile:90: *** BOARD not defined.  Stop
 ```
 
+The supported protocols are currently BLE and BOTH (ANT + BLE).
+
+### Building with an ANT softdevice
+
+Currently, the bootloader can be built against the dualstack softdevice headers for
+use of both ANT and BLE simultaneously. To do this:
+
+1. Download either s332 or s340 from thisisant.com. Note that this softdevice is
+   freely available for evaluation use only. Garmin Canada must be contacted to obtain
+   ANT licenses for commercial use.
+2. Place the contents of the softdevice package in the appropriate lib/softdevice folder.
+3. Rename the API folder to <SD name>_nrf52_6.1.1_API.
```

Comment:
can you tell me what is the default name of the s332 and s340 name, I would prefer to have the makefile to use their default name from the extracted instead.